tracker-kinect-face/camera_kinect_ir.cpp

```diff
@@ -84,7 +80,7 @@ bool CameraKinectIr::is_open()
 void CameraKinectIr::WaitForFirstFrame()
 {
     bool new_frame = false;
-    int attempts = 100;
+    int attempts = 200; // Kinect cold start can take a while
```

Comment:
That's at least 10-20 seconds in either case. Following that, it takes at least 20 seconds to start/stop a tracker in the worst case, with the UI freezing.

Reply:
Alternately you could use `compat/timer` rather than measuring attempt count. Use a minimum attempt count and a minimum time period.

Reply:
The issue here is that the PT tracker won't terminate in its destructor, freezing the entire app. Ideally the program's logic could return false in `get_frame`, returning true in `start` unconditionally.